### ISSUE TITLE
test image 

### ISSUE THREAD

**Issue by LiliMeng** [ISUE/relocforests#11]
Hi,

     Thanks a lot for the code and your helpful answer!

     Is the test image used for training in your code? It seems that one image (  Eigen::Affine3d pose = forest->Test(data->GetRGBImage(200), data->GetDepthImage(200));
>poses_eigen_.at(200))from all the data was used for testing, while all the data was used for training the regression forest.
     
    
    Thanks again!

**Comment by connerbrooks** [comment]
Yes that is correct, you are testing with a training image. That is a holdover from early development and should be resolved in the latest commits to master. I have added clarification for the example applications interface and fixed issues with the command line interface. There is now a better distinction between train and test time.

Test time statistics is still something I have to do.

**Comment by LiliMeng** [comment]
Thanks a lot! Have you tested the 7 scenes dataset? I cannot get the 31% accuracy for Chess before the optimization mentioned in the 4.4 Main Results part of the paper  either from your code and my code.

Thanks again!

**Comment by connerbrooks** [comment]
I haven't tested with any of the 7 scenes data sets. What kind of accuracy are you getting?

**Comment by LiliMeng** [comment]
Currently almost no data could be within 5 degree rotation and 5cm translation. I'll let you know the final result as soon as I complete all the test:)

**Comment by connerbrooks** [comment]
I think the issue with the 7-scenes may be due to an improper camera calibration. I currently [hard code](https://github.com/ISUE/relocforests/blob/master/source/main.cpp#L33) the camera calibration for the TUM datasets. Implementing #6 may make this more clear.

**Comment by LiliMeng** [comment]
Thanks a lot! That may make sense!
As the 7 Scenes data set use the following default intrinsics: Principle point (320,240), Focal length (585,585).
It's very different from the TUM dataset: Principle point (319.5, 239.5), focal length (525.0, 525.0)
